.gitignore

```diff
@@ -1,4 +1,8 @@
-s3watcher/node_modules
+S3Watcher/node_modules
```

Comment:
Any reason to have these ignore rules on the project root rather than, for example, in the S3Watcher directory?